main.go

```diff
@@ -135,7 +135,7 @@ func registryCommand(ctx context.Context, args *RegistryCmd) (err error) {
 		return state.Track(ctx, ociClient, router)
 	})
 
-	reg, err := registry.NewRegistry(ctx, args.RegistryAddr, ociClient, router)
+	reg, err := registry.NewRegistry(ctx, args.RegistryAddr, ociClient, router, 3)
```

Comment:
nitpick: Is it possible to allow to configurable retries instead of defaulting to 3?